backend/src/Server.ts

```diff
@@ -1,15 +1,39 @@
-import * as express from "express";
-import * as path from "path";
+import * as express      from "express";
```

Comment:
No need to align everything with spaces. It prevents you from using auto-format which is a very nice IDE feature to have. Also: https://vote4code.com/5730450056151040/aligning-require-statements